kornia/augmentation/augmentation.py

```diff
@@ -472,8 +482,20 @@ def apply_transform(
             input, transform, (height, width),
             mode=self.resample.name.lower(), align_corners=self.align_corners)
 
+    def inverse_transform(
+        self, input: torch.Tensor, transform: Optional[torch.Tensor] = None,
+        size: Optional[Tuple[int, int]] = None, **kwargs
+    ) -> torch.Tensor:
```

Comment:
could you add some docs here ? what are the kwarg ? could you make it explicit ?

Reply:
I was thinking that if it is needed to overwrite some of the parameters during the inverse conversion.

Say, I used bilinear for the forward but I may want to use nearest for inverse? I have not decided do I need to do some overwriting here. I need some real-world feedback.